dev-tools/ecs-migration.yml

```diff
@@ -6,44 +6,46 @@
 #
 # - from: source-field-in-6.x
 #   to: target-filed-in-ECS
-#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
-#   alias: true-if-alias-is-required-in-6x
+#   # Alias field is useful for fields where many-11 mapping from new to old are needed
+#   alias: true-if-alias-is-required-in-7
+#   # Alias6 field is useful for fields where there is a 1-1 mapping from old to new
+#   alias6: true-if-alias-is-required-in-6
 #   # Copy to is useful for fields where multiple fields map to the same ECS field
 #   copy_to: true-if-field-should-be-copied-to-target-in-6x
```

Comment:
Please specify that if `alias6` is absent, it's equivalent to `alias6: false`

Reply:
changed